front/src/modules/trainschedule/components/ManageTrainSchedule/helpers/buildPacedTrainException.ts

```diff
@@ -173,3 +175,126 @@ export default function updatePacedTrainException(
     ? replaceElementAtIndex(originalPacedTrain.exceptions, exceptionIndex, exception)
     : removeElementAtIndex(originalPacedTrain.exceptions, exceptionIndex);
 }
+
+/**
+ * This function is after updating a paced train when the user send the form.
+ * It checks if an exception change group can be removed.
+ * If the paced train matches an exception change group value and if it was the last one remaining,
+ * the exception is removed.
+ */
+export function checkChangeGroups(
+  originalExceptions: PacedTrainException[],
+  updatedPacedTrain: PacedTrain
```

Comment:
very nit: I find it more natural to give the updatedPacedTrain before the originalExceptions (but it's very nit)
```suggestion
  updatedPacedTrain: PacedTrain,
  originalExceptions: PacedTrainException[]
```

Reply:
Done